Automattic/jetpack
# VideoPress: Save `title`, `description` and `caption` on `Edit Details` page

Connect metadata part of `Edit Details` page with backend.

Closes #26154 

#### Changes proposed in this Pull Request:

* Extract `useMetaUpdate` from `block` to be reused.
* Create `useEditDetails` hook.
* Insert loading at `Save Changes` button.
* Introduce new `SET_VIDEO` action.

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Start `VideoPress`.
* Navigate to `dashboard` and click `Edit Details`.
* Edit video `title`.
* It should activate the `Save Changes` button.
* Click it
* It should enter in `Loading` mode.
* You will be redirected to `/`
* You should see the new title implemented.
* Reload the page.
* Title should to stay the new one.

#### Demo

https://user-images.githubusercontent.com/1663717/190449622-7aafa4c3-77a0-456f-8c57-3c5042e5e37f.mov
